mne/viz/topomap.py

```diff
@@ -916,6 +916,15 @@ def plot_ica_components(ica, picks=None, ch_type=None, res=64,
     -------
     fig : instance of matplotlib.pyplot.Figure or list
         The figure object(s).
+
+    Notes
+    -----
+    When run in interactive mode ``plot_ica_components`` allows to reject
+    components by clicking on their title label. The state of each component
+    is signalled with label color: gray - rejected; black - retained. It is
+    also possible to open component properties by clicking on the component
+    topomap. This option is only available when an mne object (Raw or Epochs)
```

Comment:
Do we capitalize MNE?

Reply:
it depends if you refer to the pypi package or the project name. See what is the most consistent with the rest of the package.

Reply:
In this case I'd use MNE here. Maybe reformulate and shorten to avoid this altogether (because the `inst` argument is documented separately anyway):

    When run in interactive mode, ``plot_ica_components`` allows to reject
    components by clicking on their title label. The state of each component
    is indicated by its label color (gray: rejected; black: retained). It is
    also possible to open component properties by clicking on the component
    topomap (this option is only available when the ``inst`` argument is
    supplied).

Reply:
👍